Leibit.Core/Data/sued/RSI.xml

```diff
@@ -0,0 +1,330 @@
+      <alternative track="5"/>
+      <alternative track="6"/>
+    </track>
+    <track name="4">
+      <block name="56G4"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="1"/>
+      <alternative track="5"/>
+      <alternative track="6"/>
+    </track>
+    <track name="5">
+      <block name="56G5"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+      <alternative track="1"/>
+      <alternative track="6"/>
+    </track>
+    <track name="6">
+      <block name="56G6"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+      <alternative track="5"/>
+      <alternative track="1"/>
+    </track>
+    <track name="41">
+      <block name="56G41"/>
+      <alternative track="42"/>
+      <alternative track="43"/>
+      <alternative track="44"/>
+      <alternative track="45"/>
+      <alternative track="46"/>
+    </track>
+    <track name="42">
+      <block name="56G42"/>
+      <alternative track="41"/>
+      <alternative track="43"/>
+      <alternative track="44"/>
+      <alternative track="45"/>
+      <alternative track="46"/>
+    </track>
+    <track name="43">
+      <block name="56G43"/>
+      <alternative track="42"/>
+      <alternative track="41"/>
+      <alternative track="44"/>
+      <alternative track="45"/>
+      <alternative track="46"/>
+    </track>
+    <track name="44">
+      <block name="56G44"/>
+      <alternative track="42"/>
+      <alternative track="43"/>
+      <alternative track="41"/>
+      <alternative track="45"/>
+      <alternative track="46"/>
+    </track>
+    <track name="45">
+      <block name="56G45"/>
+      <alternative track="42"/>
+      <alternative track="43"/>
+      <alternative track="44"/>
+      <alternative track="41"/>
+      <alternative track="46"/>
+    </track>
+    <track name="46">
+      <block name="56G46"/>
+      <alternative track="42"/>
+      <alternative track="43"/>
+      <alternative track="44"/>
+      <alternative track="45"/>
+      <alternative track="41"/>
+    </track>
+  </station>
+
+  <station name="Böhringen-Rickelshausen" short="RBER" refNr="56" scheduleFile="böhri___.abf">
+    <track name="2" calculateDelay="false">
+      <block name="56B1"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="56BP"/>
+    </track>
+  </station>
+
+  <station name="Singen (Hohentwiel) Industriegebiet" short="RSII" refNr="55" scheduleFile="b55_____.abf">
+    <track name="2" calculateDelay="false">
+      <block name="55BN1"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="55B2"/>
+    </track>
+  </station>
+  
+  <station name="Singen (Hohentwiel)" short="RSI" refNr="55" scheduleFile="g55_____.abf" localOrderFile="bf55____.abf">
+    <track name="1">
+      <block name="55G1"/>
+    </track>
+    <track name="51">
+      <block name="55G1"/>
```

Comment:
Der Block-Name passt nicht.

Reply:
Copy'n'Paste... wird gefixt.

Reply:
Bei Gleis 1 und 51 ist aber das gleiche Thema wie in Konstanz? Das ist wohl auch nur zur hälfte nutzbar.